components/atoms/Select/limit-select.tsx

```diff
@@ -6,13 +6,17 @@ import { Select, SelectContent, SelectItem, SelectTrigger, SelectValue } from "c
 interface LimitSelectProps {
   placeholder: string;
   error?: string;
+  defaultValue?: limitSelectMap;
   onChange?: (value: string) => void;
   options: { name: string; value: string }[];
   className?: string;
 }
-const LimitSelect = ({ onChange, options, className, placeholder }: LimitSelectProps) => {
-  const [selected, setSelected] = useState<null | string>(null);
-  const handleSelected = (value: string) => {
+
+export type limitSelectMap = "10" | "20" | "30" | "40" | "50";
```

Comment:
@nickytonline there are a few places within the API where we default to some set limit. Example:

https://github.com/open-sauced/api/blob/24892059033ee8e9855517d887475cdb54f8681f/src/user/dtos/top-users.dto.ts#L25-L36

But as far as I can tell, these numbers are arbitrary. The upper limit on many of these are intended to prevent full table scans.